modules/bmc/redfish.rb

```diff
@@ -0,0 +1,239 @@
+require 'redfish_client'
+require 'bmc/base'
+require 'bmc/redfish/dell'
+require 'bmc/redfish/hpe'
+
+module Proxy
+  module BMC
+    class Redfish < Base
+      include Proxy::Log
+      include Proxy::Util
+
+      def initialize(args)
+        @redfish_verify_ssl = Proxy::BMC::Plugin.settings.redfish_verify_ssl
+        raise ::Proxy::Error::ConfigurationError.new("redfish_verify_ssl must be boolean (true/false)") unless [true, false].include?(@redfish_verify_ssl)
```

Comment:
It would be cleaner to do this in a validator. In 8b27dabfcbcd6a1e8873c7f75b90a253aeb27bec I added the URL validator. You can implement a generic bool test in the same way. It also had an example of a custom validator that you can use. The benefit is that it's validated on application start up rather than on the first API request.

I would recommend to write a test for it because it's too easy to mess up otherwise.